### PR TITLE
Add COManage registration doc (SOFTWARE-4578)

### PR DESCRIPTION
**Pull request by brianhlin**
1.  Not in the ToC for now as we start the campaign to have folks register
2.  Bonus commit to update the privacy notice in the main registration page

### CODE REVIEW

**Comment by brianhlin** [comment]
Force pushed to rename the doc from `contact.md` -> `contact-registration.md`

File: docs/common/contact-registration.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+1.  Visit <https://osg-htc.org/register>
+
+1.  You will be presented with a Single-Sign On page.
+    Select your insitution and sign in with your insitutional credentials:
+
+    !!! question "Help, my institution does not show up in the drop-down!"
+        If your institution does not show up in the drop-down menu, then your institution is not part of the
+        [InCommon federation](https://www.incommon.org/federation/).
+        Instead, we recommend using an [ORCID account](https://orcid.org/), registering a new one if necessary.
```
Comment (edquist) [inline docs/common/contact-registration.md:32]: To my eye, the leading "Instead" makes it sound unclear whether we _always_ recommend using an OCRID account instead, or just in the case that your institution does not show up.

Does this read any better for you?

```suggestion
        In this case, we recommend using an [ORCID account](https://orcid.org/) instead, registering a new one if necessary.
```

File: docs/common/registration.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ associated GitHub ID.
 To register as an OSG contact or add your GitHub ID to your pre-existing contact entry, search for your name in the
 [contact database](https://topology.opensciencegrid.org/contacts) and choose one of the following:
 
+!!! info "Privacy"
+    The OSG treats any email addresses and phone numbers as confidential data but does not make any guarantees of
+    privacy.
+    All other data is public, such as name, GitHub username, and any association with particular hosts or services.
```
Comment (edquist) [inline docs/common/registration.md:35]: :+1:

**Review by edquist — APPROVED** [review]
One small tweak otherwise lgtm

**Pull request merged**